features/backend/countries.feature

```diff
@@ -69,7 +69,8 @@ Feature: Countries and provinces
     @javascript
     Scenario: Creating new country with provinces
         Given I am on the country creation page
-         When I fill in "Name" with "Poland"
+         When I click "En_us"
```

Comment:
Is there a better way to describe or handle this? Normally a locale would be set as the default, but when there is none, the form field for "Name" is not visible until you click on the locale tab.